ide/spi.debugger.ui/src/org/netbeans/spi/debugger/ui/DebuggingView.java

```diff
@@ -401,6 +444,51 @@ public final void removePropertyChangeListener(PropertyChangeListener pcl) {
         public DVThreadGroup[] getThreadGroups();
     }
     
+    /**
+     * Representation of a stack frame in debugging view.
+     * Nodes representing a stack frame in debugging view model should implement this
+     * interface.
```

Comment:
Missing `@since` tag.

Reply:
Thanks for the reminder of the apichanges, etc. Updated.